src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/HandScoreTest.java

```diff
@@ -0,0 +1,218 @@
+            .withPlayedCard(PLAYER_3, ACE_OF_SPADES)
+            .finishRound()
+            .withRound()
+            .withPlayedCard(PLAYER_1, JACK_OF_CLUBS)
+            .withPlayedCard(PLAYER_2, EIGHT_OF_CLUBS)
+            .withPlayedCard(PLAYER_3, QUEEN_OF_SPADES)
+            .finishRound()
+            .finishMatch()
+            .build();
+
+    game.getLastMatch()
+        .flatMap(Match::getLastRound)
+        .ifPresent(lastRound -> lastRound.setRoundNumber(9));
+    gameRepository.saveAll(List.of(game));
+
+    Match match = game.getSortedMatches().get(0);
+    List<Hand> sortedHands = match.getSortedHands();
+    Hand handPlayer1 = sortedHands.get(0);
+    Hand handPlayer2 = sortedHands.get(1);
+    Hand handPlayer3 = sortedHands.get(2);
+
+    assertThat(handPlayer1.getNumberOfWonTricks(), is(ANNOUNCED_SCORE_PLAYER_1));
+    assertThat(handPlayer1.getPoints(), is(ANNOUNCED_SCORE_PLAYER_1 * ANNOUNCED_SCORE_PLAYER_1));
+
+    assertThat(handPlayer2.getNumberOfWonTricks(), is(ANNOUNCED_SCORE_PLAYER_2));
+    assertThat(handPlayer2.getPoints(), is(ANNOUNCED_SCORE_PLAYER_2));
+
+    assertThat(handPlayer3.getNumberOfWonTricks(), is(2));
+    assertThat(handPlayer3.getPoints(), is(-Math.abs(ANNOUNCED_SCORE_PLAYER_3 - 2)));
+  }
+
+  @Test
+  void evaluates_score_with_multiple_highest_cards() {
+    Game game =
+        matchBuilderWithHands
+            .withRound()
+            .withPlayedCard(PLAYER_1, ACE_OF_CLUBS)
+            .withPlayedCard(PLAYER_2, KING_OF_CLUBS)
+            .withPlayedCard(PLAYER_3, ACE_OF_SPADES)
+            .finishRound()
+            .withRound()
+            .withPlayedCard(PLAYER_1, SEVEN_OF_CLUBS)
+            .withPlayedCard(PLAYER_2, JACK_OF_SPADES)
+            .withPlayedCard(PLAYER_3, QUEEN_OF_CLUBS)
+            .finishRound()
+            .withRound()
+            .withPlayedCard(PLAYER_1, JACK_OF_CLUBS)
+            .withPlayedCard(PLAYER_2, EIGHT_OF_CLUBS)
+            .withPlayedCard(PLAYER_3, QUEEN_OF_SPADES)
+            .finishRound()
+            .finishMatch()
+            .build();
+    game.getLastMatch()
+        .flatMap(Match::getLastRound)
+        .ifPresent(lastRound -> lastRound.setRoundNumber(9));
+    gameRepository.saveAll(List.of(game));
+
+    Match match = game.getSortedMatches().get(0);
+    List<Hand> sortedHands = match.getSortedHands();
+    Hand handPlayer1 = sortedHands.get(0);
+    Hand handPlayer2 = sortedHands.get(1);
+    Hand handPlayer3 = sortedHands.get(2);
+
+    assertThat(handPlayer1.getNumberOfWonTricks(), is(0));
+    assertThat(handPlayer1.getPoints(), is(-ANNOUNCED_SCORE_PLAYER_1));
+
+    assertThat(handPlayer2.getNumberOfWonTricks(), is(ANNOUNCED_SCORE_PLAYER_2));
+    assertThat(handPlayer2.getPoints(), is(ANNOUNCED_SCORE_PLAYER_2));
+
+    assertThat(handPlayer3.getNumberOfWonTricks(), is(ANNOUNCED_SCORE_PLAYER_3));
+    assertThat(handPlayer3.getPoints(), is(ANNOUNCED_SCORE_PLAYER_3 * ANNOUNCED_SCORE_PLAYER_3));
+  }
```

Comment:
Wie wärs mit noch einem Test in dem zwei Spieler die höchste Karte spielen --> Stack --> in der nächsten Runde räumt der dritte Spieler ab (der in der Runde zuvor die tiefste Karte hatte).
Können auch ein Issue machen und später noch hinzufügen.

Reply:
Gibt es schon #31